src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1282,6 +1282,7 @@
   "com.minecolonies.coremod.general.canttakeequipped": "The only armor you have is what you're wearing. Put it in your inventory if you want to give it to me.",
 
   "com.minecolonies.coremod.gui.workerhuts.siftersettings": "Daily limit",
+  "com.minecolonies.coremod.gui.workerhuts.sifterinfo": "Max %d :",
```

Comment:
```suggestion
  "com.minecolonies.coremod.gui.workerhuts.sifterinfo": "Max %d",
```
If the max per day is before the colon, there's no need for it.

Reply:
So, this displays as "Max %d : <textbox with>", so at level 1 it would be "Max 64 : [   0]" by default.

Reply:
Where the 64 is the max and the 0 is the current setting? I'd do something like `Daily Output 0/64` then

Reply:
Well, the 0 is an input box so hard to put in the middle of a string like that. I expect this whole area to change further with Phase 2, possibly even remove this portion of the UI altogether.

Reply:
Ahh it's an input box. Two lines possibly?
`"com.minecolonies.coremod.gui.workerhuts.sifterinfo1": "Max: %d",
`"com.minecolonies.coremod.gui.workerhuts.sifterinfo2": "Current: [fancy infobox stuff here]",

Reply:
Here's what it looks like on my test world: 
![Imgur](https://i.imgur.com/8vDrrYfm.png)

Reply:
Kinda looks like you're changing the max imo

Reply:
Ok, Updated to look like this: 
![Imgur](https://i.imgur.com/qcOhkUtm.png)

Reply:
👍